ide-configuration/eclipse-format.xml

```diff
@@ -1,323 +1,322 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <profiles version="15">
-<profile kind="CodeFormatterProfile" name="Quarkus" version="15">
```

Comment:
Sorry about changing the entire file due to indentation. Let me mark lines where the value changed.